reference/functions/useringroup.markdown

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+title: useringroup
+published: true
+---
+
+[%CFEngine_function_prototype(user)%]
+
+**Description:** Return whether a `user` is a member of a given `group`
+
+Looks up the group `"group_name"` in the group database located in /etc/group and checks whether `"user_name"` is one of its members.
```

Comment:
Will this function work on Windows? If so, then the group database is probably not located in /etc/group